docs/install/scala_setup.md

```diff
@@ -0,0 +1,87 @@
+# Setup the MXNet Package for Scala
```

Comment:
Build MXNet Package for Scala from source

Reply:
That's not the main purpose of this page. It is to setup your system to use the maven package, and it also has some building from source information **provided for reference only**. 

The only reason I had to add the git clone step was because I thought the user was going to need to run example or demo code from the repo in Step 3.

Otherwise for the dependencies I would have had the user just wget the CI scripts.